URLMock/Categories/NSURL+UMKQueryParameters.h

```diff
@@ -83,6 +89,10 @@
  @param baseURL The base URL for the NSURL object.
  @result An NSURL object
  */
-+ (instancetype)umk_URLWithString:(NSString *)URLString parameters:(NSDictionary *)parameters relativeToURL:(NSURL *)baseURL;
++ (_Nullable instancetype)umk_URLWithString:(NSString *)URLString
+                                 parameters:(NSDictionary * _Nullable)parameters
+                              relativeToURL:(NSURL * _Nullable)baseURL NS_SWIFT_NAME(init(string:parameters:relativeToURL:));
```

Comment:
I got this from Apple’s documentation here:

https://developer.apple.com/library/ios/documentation/Swift/Conceptual/BuildingCocoaApps/MixandMatch.html#//apple_ref/doc/uid/TP40014216-CH10-ID167

So, I assume it’s okay…